zerver/views/custom_profile_fields.py

```diff
@@ -44,6 +45,7 @@ def list_realm_custom_profile_fields(
 
 hint_validator = check_capped_string(CustomProfileField.HINT_MAX_LENGTH)
 name_validator = check_capped_string(CustomProfileField.NAME_MAX_LENGTH)
+max_display_in_profile_summary_fields = 2
```

Comment:
Moved this constant to be a property of `CustomProfileField` and be capitalized, for better consistency with our style.